reporting-guide.md

```diff
@@ -440,26 +443,35 @@ Here, I'm requesting `evar1`. In this example, the evar is simply storing a nume
 }
 ```
 
-Let's look in detail at one of the records returned:
-```json
-{
-   "itemId":"743855946",
-   "value":"10",
-   "data":[
-      2032.0
-   ]
-},
-```
+## Filtering Reports
 
-* `itemId` - This is the unique ID associated with this particular value - in this case, campaign "10".
-* `value` - This contains the value of the evar.
-* `data` - This is an array of counts - one for each metric requested.
+Use filters to limit the data returned so that reports show only the values you need.  For example, if you have thousands of records but only a few have needed reports, you can use filtering to return and find them quickly. Some filters also allow you to include, group, or present data in convenient formats, such as breakdown reports.
 
-#### Filters
-In the above example, only the top 5 records were requested and there were 8 pages worth of data available -- only 40 records. What if there were thousands of records, and only a handful were being reported on? Filters can be used to only include those few valuable values in the report.
+This section includes information on the following filtering features:
 
-For the next example, let's only request data for campaigns "10" and "11" by using their itemIds.
-```json
+* [Search parameter](#search)
+
+* [Clause parameter](#clause)
+
+* [Applying Segments](#segments)
+
+* [Breakdown Reports](#breakdown)
+
+### <a name="search">Using `search` Parameters</a>
```

Comment:
Do you really want this header to be a hyperlink? They don't render well and it doesn't look like it actually goes anywhere.

Reply:
These are midtopic links and have worked well so far on former Adobe IO renderings. They are very helpful for layout navigation and are basic documentation tools. Take a look at Adobe Help on Analytics and you will find them everywhere. (https://marketing.adobe.com/resources/help/en_US/analytics/getting-started/analytics-navigation.html) Basically, if you have a list of what is covered in a section, you link it to the section. They conform to the instructions provided by Adobe IO "doc collaboration model." If for some reason they don't work, they can be removed. (I know they look a little backward because of where the HTML anchor sits but that's the way it works with markdown.)

Reply:
The QA team for adobe.io asked us to remove the last tags like this that you had added since they make the cursor look different on hover, but didn't provide any functionality. We can try these out and see if they behave any better this time.

Reply:
Ok. I'm not sure which tags you are referring to. I have four like this. Are you saying we should remove all four? Is it possible to find out who I can talk to on Adobe.io QA because that might help me a lot. They must have a way to allow midtopic links. How do you see them render on Adobe IO before they are put on Adobe IO? I should probably be using that tool--whatever it is--and it would work much better for me.

Reply:
Ill merge this change and we can see how it works on adobe.io. If it is functional then we will be in good shape. If not we can remove the anchor tags and look for a different way to do what your looking for